packages/cactus-plugin-ledger-connector-besu/src/main/typescript/plugin-ledger-connector-besu.ts

```diff
@@ -160,6 +160,14 @@ export class PluginLedgerConnectorBesu
       endpoint.registerExpress(expressApp);
       endpoints.push(endpoint);
     }
+    {
```

Comment:
@AzaharaC I think this might've become redundant after a rebase (there's already `InvokeContractEndpointV2` registered in the same function)